### ISSUE TITLE
The file "PushDesktop.ax" is missing

### ISSUE THREAD

**Issue by ldlshizhu** [rdp/screen-capture-recorder-to-video-windows-free#120]
Hello, thank you very much for your project, I have successfully recorded the screen. However, I want to run the source code, but I find that the file "PushDesktop.ax" is missing and cannot run. Could you please provide this file? Thank you ！

**Comment by rdp** [comment]
I think it's renamed to something.dll these days...

**Comment by ldlshizhu** [comment]
Yes, I have seen it. It seems to be related to the file ”screen-capture-recorder. dll”.However, I forced the file to be renamed "pushdesktop.ax" and the file was invalidated (obviously). Could you please sort out the files of this project? Thank you very much for your contribution.

**Comment by rdp** [comment]
You just want the readme to be made accurate?

**Comment by ldlshizhu** [comment]
I'm sorry I didn't make myself clear. I would like to run this project on vs, but there are some missing files you uploaded on github. Please kindly provide the complete project file.thank you

**Comment by rdp** [comment]
Hmm I think I'm able to compile it by opening the sln files here https://github.com/rdp/screen-capture-recorder-to-video-windows-free/tree/master/source_code what is missing?

**Comment by ldlshizhu** [comment]
> Hmm I think I'm able to compile it by opening the sln files here https://github.com/rdp/screen-capture-recorder-to-video-windows-free/tree/master/source_code what is missing?

I had downloaded the source code zip, but the "pushdesktop.ax" file was not in the zip. The reason why I insisted on this file was that when I tried to run this project in vs, I was always prompted that "pushdesktop.ax" was missing and thus failed. My friends also had the same problem. Maybe you can download the project from this web page and run it again. I guess some changes in the last year made the files on this website changed without your realizing. So please review the source code of this project and upload it. Thank you very much.

**Comment by rdp** [comment]
These days I mostly release .exe's was it an old file? Which file did you
download and how did you try and open it in vs?  Sorry for the confusion.

**Comment by rdp** [comment]
Hopefully we can get it figured out, if you download the git repo the sln's
should work with VS express 2010 at least, or were you downloading virtual
audio capturer?

**Comment by ldlshizhu** [comment]
I'm in the url "https://github.com/rdp/screen-capture-recorder-to-video-windows-free", click "clone or download" button to download the zip. If this project runs successfully, it should record the screen and store the effect, right? My system is win7 64-bit and I used vs2015 to open E:\360Downloads\ screen-captup-recorder to video-windows-free-master\source_code\ pushdesk.x64.sln and made a running error. Do you think there is any problem with the above operation? Thank you for your patience.

**Comment by ldlshizhu** [comment]
The url "https://github.com/rdp/screen-capture-recorder-to-video-windows-free/commit/e82d7c4b69e8b79758e468724d518bdbf1178325" seems to be written : deleted "pushdesktop. ax," added "screen - the capture - recorder. dll. "I feel like it's not working because of this change.

**Comment by rdp** [comment]
OK normally people will want to download the .exe installer, but I presume
you want the source in this instance? If yes what running error does VS
give?  Thanks!

**Comment by ldlshizhu** [comment]
“PushDesktop.ax”is missing！hahaha

**Comment by ldlshizhu** [comment]
umm Don't you have this file(pushDesktop.ax) in your project? Could you find it on your computer? Thank you very much

**Comment by Lexian-droid** [comment]
https://github.com/rdp/screen-capture-recorder-to-video-windows-free/blob/master/source_code/win32/Release/screen-capture-recorder.dll isn't this it? It seems renamed. I need it for my own tool, so I guess i'ma just.. hippity hoppity? license and readme both contradict.

**Comment by rdp** [comment]
yeah I renamed it at some point to screen-capture-recorder.dll where does
it say the old name?

**Comment by Lexian-droid** [comment]
> yeah I renamed it at some point to screen-capture-recorder.dll where does
> it say the old name?
> […](#)

In the readme and license

**Comment by rdp** [comment]
I didn't see it in the license, but hopefully got the README now, thanks!
